src/Components/Rocket.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { makeReservation, cancelReservation } from '../Redux/rockets/rocketsSlice';
+
+const Rocket = ({
+  name, description, image, id, reserved,
+}) => {
+  const dispatch = useDispatch();
```

Comment:
- Kindly, you are dispatching the state when you navigate to the page which is resetting(clearing the reserved rockets) the state, please dispatch the action for the first time only using `useEffect` as you did for missions 👍 to have the correct data.
- Kindly use also `useSelector` for getting the updated state from the store as you did in missions 👌

Reply:
Resolved: using useSelector and maintaining state after the first API fetch